src/main/java/com/redhat/devtools/lsp4ij/features/inlayhint/LSPInlayHintsProvider.java

```diff
@@ -61,7 +61,7 @@ protected void doCollect(@NotNull PsiFile psiFile,
 
         try {
             // Wait until the future is finished and stop the wait if there are some ProcessCanceledException.
-            waitUntilDone(future, psiFile);
+            waitUntilDone(future, psiFile, 25);
```

Comment:
Is 25 ms is enough? Perhaps we should set 500 ms to cover cases of ls which give  a fast LSP response?